Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -2771,6 +2771,45 @@ bool CSSPropertyParser::consumeWhiteSpaceShorthand(bool important)
     return true;
 }
 
+
+bool CSSPropertyParser::consumeAnimationRangeShorthand(bool important)
+{
+    // TODO: handle animation range sequences
```

Comment:
Our project uses `FIXME` not `TODO`. And we write comments sentence style with a period and a capital letter.

Reply:
Fixed.